Devices/Lumia/950s/Talkman/Main.txt

```diff
@@ -0,0 +1,185 @@
+	InstallDevMenu();
+	PrepareDisk();
+	DeployWindows();
+	AddDrivers();
+	InstallBootShim();
+	InstallUefi();	
+}
+
+Common
+{
+	packages = "Packages";
+	content =   "..\..\Content";
+
+	efiEspPart = "Disk={Disk}, Name='EFIESP'";
+	efiEspRoot = GetPartitionRoot("{efiEspPart}");
+}
+
+ShowReadme
+{
+	Unzip(GitHubLatestReleaseAsset("WOA-Project", "Lumia-Drivers", "Changelog.zip"), packages);
+	DisplayMarkdown("{packages}\Changelog\changelog.md");
+}
+
+InstallDevMenu
+{
+	'Installing Developer Menu'
+	
+	CopyDirectory("{content}\Developer Menu", "{efiEspRoot}\Windows\System32\BOOT");
+	
+	devMenuGuid = "7619dccc-fafe-11d9-b411-000476eba25f";
+	efiBcdStore = "{efiEspRoot}\EFI\Microsoft\Boot\BCD";
+	
+	BcdEditCreate(efiBcdStore, "{devMenuGuid}", "/d ""Developer Menu"" /application BOOTAPP");
+	BcdEdit(efiBcdStore, "/set {{{devMenuGuid}}} path \Windows\System32\BOOT\developermenu.efi");
+	BcdEdit(efiBcdStore, "/set {{{devMenuGuid}}} device partition={efiEspRoot}");
+	BcdEdit(efiBcdStore, "/set {{{devMenuGuid}}} nointegritychecks on");
+	BcdEdit(efiBcdStore, "/displayorder {{{devMenuGuid}}} /addlast");
+}
+
+PrepareDisk
+{
+	'Preparing partition layout'
+	
+	RemovePartitionSafe("Disk={Disk}, Name='MSR'");
+	RemovePartitionSafe("Disk={Disk}, Name='Windows'");
+	RemovePartitionSafe("Disk={Disk}, Name='SYSTEM'");
+	
+	if (GetDiskSize(Disk) < SizeFromString("{DeploymentSize}GB"))
+	{
+		'Trying to allocate required space from the Data partition'
+
+		data = "Disk={Disk}, Name='Data'";
+		if (PartitionExists(data)) 
+		{
+			'Allocating space'
+			AllocateSpace(data, "{DeploymentSize}GB");	
+		}	
+		else 
+		{
+			Fail("Could not allocate the required space in the phone. We need at least {DeploymentSize}GB of free space to deploy Windows");
+		}
+	} 
+		
+	systemPart = CreateGptPartition(Disk, "Esp", "SYSTEM", "100MB");
+	CreateGptPartition(Disk, "Reserved", "MSR", "16MB");
+	windowsPart = CreateGptPartition(Disk, "Basic", "Windows", "*");
+	
+	systemRoot = GetPartitionRoot(systemPart);
+	windowsRoot = GetPartitionRoot(windowsPart);
+	
+	Format(systemPart, "Fat32", "SYSTEM");
+	Format(windowsPart, "Ntfs", "Windows");
+}
+
+DeployWindows
+{
+	'Deploying Windows'
+
+	ApplyWindowsImage(WimFilePath, WimFileIndex, windowsRoot);
+	MakeWindowsBootable("{systemRoot}", "{windowsRoot}\Windows");
+	SetGptType("{systemPart}", "Esp");
+}
+
+InstallBootShim
+{
+	'Downloading UEFI Loader (BootShim)'
+	Unzip(AzureDevOpsArtifact("LumiaWOA", "Boot Shim", 3, "UEFI Loader for 8992 and 8994"), "{packages}\UEFI Loader");
+
+	'Installing UEFI Loader'
+	Copy("{packages}\UEFI Loader\artifacts\UEFI Loader for 8992 and 8994\ARM\Debug\BootShim.efi", "{efiEspRoot}\EFI\Boot\BootShim.efi");
+}
+
+ConfigureBcd
+{
+	'Configuring Boot'
+	
+	bcdWoa = "7619dcca-fafe-11d9-b411-000476eba25f";
+	bcdWinMobile = "7619dcc9-fafe-11d9-b411-000476eba25f";
+	efiBcdStore = "{efiEspRoot}\EFI\Microsoft\Boot\BCD";
+	
+	BcdEditCreate(efiBcdStore, bcdWoa, "/d ""Windows 10"" /application BOOTAPP");
```

Comment:
(Nitpick) Would probably change this to "Windows Desktop" or "Full Windows" or "Windows on ARM" considering Windows 11 and other things)

Reply:
I'm make this change in the next PR, thank you mate!